### DISCUSSION TITLE
Potential Renpy support

### DISCUSSION THREAD

**Discussion by AceOfSpades719** [OnionUI/Onion#1348] [Ideas]
I was told that Renpy emulation was potentially "doable" and to "fire a suggestion" in the github discussions. Considering the usual lack of heavy effects and relatively simple gameplay, I believe Renpy shouldn't be to hard on the Miyoo

**Comment by XK9274** [comment]
I'll look into this to see if it's possible, thanks for dropping it in

**Reply by Drago-Cuven** [reply]
any updates?
